README.md

```diff
@@ -61,8 +61,10 @@ clear                          Start from a fresh install (needed if vendors hav
 
 Use double-dash to pass any PHPUnit options or arguments with `make`:
 ```shell
-$ make test -- --filter=FactoryTest
 $ make test -- --stop-on-failure
+$ make test -- --filter FactoryTest
+# don't use "=" options value. ie: don't do this:
+$ make test -- --filter=FactoryTest
```

Comment:
I've struggled with this, and discovered `make test -- --filter=FactoryTest` won't work although the other way works
sometimes Makefiles works in mysterious ways  :shrug: